api/PaymentRequest.json

```diff
@@ -604,36 +605,35 @@
           }
         }
       },
-      "requestId": {
+      "shippingAddress": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentRequest/requestId",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentRequest/shippingAddress",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "61"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "53"
             },
             "edge": {
-              "version_added": "≤18",
-              "version_removed": "79"
+              "version_added": "15"
             },
             "firefox": {
               "version_added": false,
-              "notes": "Available only in nightly builds."
+              "notes": "Available only in nightly builds as <code>shippingAddress</code>."
             },
             "firefox_android": {
               "version_added": false,
-              "notes": "Available only in nightly builds."
+              "notes": "Available only in nightly builds as <code>shippingAddress</code>."
             },
             "ie": {
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "version_added": "47"
             },
             "opera_android": {
-              "version_added": false
+              "version_added": "44"
             },
             "safari": {
               "version_added": null
```

Comment:
Safari support should probably be 11.1 (the version that first enabled Payment Request), as it seems it has have the attribute since the beginning.

https://webkit-search.igalia.com/webkit/source/Source/WebCore/Modules/paymentrequest/PaymentRequest.idl

Reply:
> Safari support should probably be 11.1 (the version that first enabled Payment Request), as it seems it has have the attribute since the beginning.
> 
> https://webkit-search.igalia.com/webkit/source/Source/WebCore/Modules/paymentrequest/PaymentRequest.idl

Thanks — will raise a separate PR for that